src/Illuminate/Routing/Router.php

```diff
@@ -1228,4 +1230,20 @@ public function getPatterns()
     {
         return $this->patterns;
     }
+
+    /**
+     * Dynamically handle calls into the router instance.
+     *
+     * @param  string  $method
+     * @param  array  $parameters
+     * @return mixed
+     */
+    public function __call($method, $parameters)
+    {
+        if (static::hasMacro($method)) {
+            return $this->macroCall($method, $parameters);
+        }
+
+        return (new RouteRegistrar($this))->attribute($method, $parameters[0]);
```

Comment:
I think this should be done like [here](https://github.com/JosephSilber/framework/blob/74ed0796e99c635699707463d5e320ff33dba671/src/Illuminate/Routing/Router.php#L275)